src/main/java/Ui.java

```diff
@@ -0,0 +1,74 @@
+import java.util.Scanner;
+
+public class Ui {
+
+    /** Scanner object used to receive inputs from the user **/
+    private Scanner sc;
+    /** Line separator to beautify duke bot **/
+    private String line = "____________________________\n"
+                        + "____________________________\n";
```

Comment:
Perhaps the variable can be final, with a capitalised variable name as a constant?